### ISSUE TITLE
Error when add Planning External Event Template

### ISSUE THREAD

**Issue by MisterSlam** [glpi-project/glpi#12676]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.3

### Bug description

When I try to add an external event template, in normal mode - empty window. In debug mode, an error appears:
![image](https://user-images.githubusercontent.com/32896480/190873393-d6732be6-8be8-41de-871d-2b9485b694c9.png)

### Relevant log output

```shell
[2022-09-17 21:59:39] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: Argument 1 passed to PlanningExternalEventTemplate::showRepetitionForm() must be of the type string, array given, called in /src/PlanningExternalEventTemplate.php on line 112 in /src/Features/PlanningEvent.php at line 724
  Backtrace :
  src/PlanningExternalEventTemplate.php:112          PlanningExternalEventTemplate::showRepetitionForm()
  .../twig/twig/src/Extension/CoreExtension.php:1607 PlanningExternalEventTemplate->displaySpecificTypeField()
  ...4d546e6780f76d5104d79a0e17a135e30177fa5.php:459 twig_get_attribute()
  vendor/twig/twig/src/Template.php:394              __TwigTemplate_4312500f5a98fda35667ca1136a2df4311ee8ca21bbcf4749b55eed511cfc15b->doDisplay()
  vendor/twig/twig/src/Template.php:367              Twig\Template->displayWithErrorHandling()
  vendor/twig/twig/src/Template.php:379              Twig\Template->display()
  vendor/twig/twig/src/TemplateWrapper.php:40        Twig\Template->render()
  src/Application/View/TemplateRenderer.php:161      Twig\TemplateWrapper->render()
  src/CommonDropdown.php:365                         Glpi\Application\View\TemplateRenderer->render()
  src/CommonGLPI.php:676                             CommonDropdown->showForm()
  ajax/common.tabs.php:113                           CommonGLPI::displayStandardTab()
```

### Page URL

/front/planningexternaleventtemplate.form.php

### Steps To reproduce

1. Settings - Dropdown - PlanningExternalEventTemplate
2. Add

### Your GLPI setup information

<details><summary>Информация о системе, установке и конфигурации</summary><pre>GLPI 10.0.3 ( => )
Installation mode: TARBALL
Current language:ru_RU

</details><details><summary>Server</summary><pre> 
Operating system: Linux vali.from.sh 3.10.0-1160.76.1.el7.x86_64 #1 SMP Wed Aug 10 16:21:17 UTC 2022 x86_64
PHP 7.4.30 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, bcmath, bz2, calendar,
	ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, geoip, gettext, gmp, hash, iconv, igbinary, imagick, imap, intl, json,
	ldap, libxml, mbstring, mcrypt, memcache, memcached, msgpack, mysqli, mysqlnd, openssl, pcre, pdo_dblib, pdo_mysql, pdo_sqlite,
	posix, redis, session, soap, sockets, sqlite3, standard, tidy, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="60" memory_limit="128M" post_max_size="100M" safe_mode="" session.save_handler="files"
	upload_max_filesize="1024M" 
Software: Apache/2.4.6 (Apache/2.4.6 Server at  Port 80
)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36 Edg/105.0.1343.33
Server Software: Percona Server (GPL), Release 40, Revision 3a1347ec0d4
	Server Version: 5.7.37-40
	Server SQL Mode: 
	Parameters: 
	Host info: Localhost via UNIX socket
	
PHP version (7.4.30) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (5.7.37) is supported.
The log file has been created successfully.
Write access to /files/_cache has been validated.
Write access to /config has been validated.
Write access to /files/_cron has been validated.
Write access to /files has been validated.
Write access to /files/_dumps has been validated.
Write access to /files/_graphs has been validated.
Write access to /files/_lock has been validated.
Write access to /files/_pictures has been validated.
Write access to /files/_plugins has been validated.
Write access to /files/_rss has been validated.
Write access to /files/_sessions has been validated.
Write access to /files/_tmp has been validated.
Write access to /files/_uploads has been validated.
Web access to files directory is protected
SELinux configuration is OK.
PHP directive "session.cookie_secure" should be set to "on" when GLPI can be accessed on HTTPS protocol.
PHP directive "session.cookie_httponly" should be set to "on" to prevent client-side script to access cookie values.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
zip extension is installed.
bz2 extension is installed.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring.
Following extensions are not present: sodium.
Write access to /marketplace has been validated.
Access to timezone database (mysql) is not allowed.

</details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: 
GLPI_CONFIG_DIR: /config
GLPI_VAR_DIR: /files
GLPI_MARKETPLACE_DIR: /marketplace
GLPI_USE_CSRF_CHECK: 1
GLPI_CSRF_EXPIRES: 7200
GLPI_CSRF_MAX_TOKENS: 100
GLPI_USE_IDOR_CHECK: 1
GLPI_IDOR_EXPIRES: 7200
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: 

**PHP Notice (8): Array to string conversion in /src/Config.php at line 2073**
GLPI_SERVERSIDE_URL_ALLOWLIST: Array
GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org
GLPI_INSTALL_MODE: TARBALL
GLPI_NETWORK_MAIL: glpi@teclib.com
GLPI_NETWORK_SERVICES: https://services.glpi-network.com
GLPI_MARKETPLACE_ALLOW_OVERRIDE: 1
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: 1
GLPI_USER_AGENT_EXTRA_COMMENTS: 
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: 1
GLPI_AJAX_DASHBOARD: 1
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: 0
GLPI_CENTRAL_WARNINGS: 1
GLPI_DOC_DIR: /files
GLPI_CACHE_DIR: /files/_cache
GLPI_CRON_DIR: /files/_cron
GLPI_DUMP_DIR: /files/_dumps
GLPI_GRAPH_DIR: /files/_graphs
GLPI_LOCAL_I18N_DIR: /files/_locales
GLPI_LOCK_DIR: /files/_lock
GLPI_LOG_DIR: /files/_log
GLPI_PICTURE_DIR: /files/_pictures
GLPI_PLUGIN_DOC_DIR: /files/_plugins
GLPI_RSS_DIR: /files/_rss
GLPI_SESSION_DIR: /files/_sessions
GLPI_TMP_DIR: /files/_tmp
GLPI_UPLOAD_DIR: /files/_uploads
GLPI_INVENTORY_DIR: /files/_inventories
GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/
GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/
GLPI_I18N_DIR: /locales
GLPI_VERSION: 10.0.3
GLPI_SCHEMA_VERSION: 10.0.3@a130db99c7d9b131c2e2ea59fe0d6260fe93d831
GLPI_MARKETPLACE_PRERELEASES: 
GLPI_MIN_PHP: 7.4.0
GLPI_MAX_PHP: 8.3.0
GLPI_YEAR: 2022

</details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.9 in (/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.6.0 in (/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.5.0 in (/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/vendor/michelf/php-markdown/Michelf)
true/punycode in (/vendor/true/punycode/src)
iamcal/lib_autolink in (/vendor/iamcal/lib_autolink)
sabre/dav in (/vendor/sabre/dav/lib/DAV)
sabre/http in (/vendor/sabre/http/lib)
sabre/uri in (/vendor/sabre/uri/lib)
sabre/vobject in (/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/vendor/elvanto/litemoji/src)
symfony/console in (/vendor/symfony/console)
scssphp/scssphp in (/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/vendor/ramsey/uuid/src)
psr/log in (/vendor/psr/log/Psr/Log)
psr/simple-cache in (/vendor/psr/simple-cache/src)
psr/cache in (/vendor/psr/cache/src)
league/csv in (/vendor/league/csv/src)
mexitek/phpcolors in (/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/vendor/paragonie/sodium_compat/src)
symfony/cache in (/vendor/symfony/cache)
html2text/html2text in (/vendor/html2text/html2text/src)
symfony/css-selector in (/vendor/symfony/css-selector)
symfony/dom-crawler in (/vendor/symfony/dom-crawler)
twig/twig in (/vendor/twig/twig/src)
twig/string-extra in (/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 in (/vendor/symfony/polyfill-php80)
symfony/polyfill-php81 in (/vendor/symfony/polyfill-php81)
symfony/polyfill-php82 in (/vendor/symfony/polyfill-php82)

</details><details><summary>SQL replicas</summary><pre> 
Not active

</details><details><summary>Notifications</summary><pre> 
Way of sending emails: SMTP+TLS ()

</details><details><summary>Plugins list</summary><pre> 
	barcode              Name: Barcode                        Version: 2.7.1      State: Enabled                                 
		Install Method: Marketplace
	datainjection        Name: Data injection                 Version: 2.10.2     State: Not installed                           
		Install Method: Marketplace
	databases            Name: Databases                      Version: 2.3.2      State: Not installed                           
		Install Method: Marketplace
	formcreator          Name: Form Creator                   Version: 2.13.0     State: Enabled                                 
		Install Method: Marketplace
	gappessentials       Name: Gapp Essentials                Version: 2.0.1      State: Enabled                                 
		Install Method: Marketplace
	glpiinventory        Name: GLPI Inventory                 Version: 1.0.4      State: Enabled                                 
		Install Method: Marketplace
	mod                  Name: GLPI Modifications             Version: 1.3.0      State: Not installed                           
		Install Method: Manual
	geninventorynumber   Name: Inventory number generation    Version: 2.5.1      State: Not installed                           
		Install Method: Marketplace
	mailanalyzer         Name: Mail Analyzer                  Version: 1.6.4      State: Not installed                           
		Install Method: Manual
	metademands          Name: Meta-Demands                   Version: 2.6.1      State: Not installed                           
		Install Method: Manual
	oauthimap            Name: Oauth IMAP                     Version: 1.4.1      State: Enabled                                 
		Install Method: Marketplace
	telegrambot          Name: TelegramBot                    Version: 3.0        State: Enabled                                 
		Install Method: Manual
	fields               Name: Дополнительные поля    Version: 1.17.3     State: Enabled                         
		        Install Method: Marketplace
	consumables          Name: Заказ расходных материалов Version: 1.4.1      State: Not installed         
		                  Install Method: Manual
	environment          Name: Окружение                      Version: 2.3.0      State: Not installed                    
		       Install Method: Manual
	news                 Name: Оповещения                     Version: 1.10.5     State: Enabled                         
		        Install Method: Marketplace
	dashboard            Name: Панель отчетов              Version: 0.9.5      State: Not installed                   
		        Install Method: Manual
	escalade             Name: Переназначение             Version: 2.8.0      State: Enabled                         
		        Install Method: Marketplace
	mreporting           Name: Подробные отчеты          Version: 1.8.1      State: Enabled                         
		        Install Method: Marketplace
	archires             Name: Сетевые архитектуры    Version: 2.7.0      State: Not installed                   
		        Install Method: Manual
	shellcommands        Name: Служебные команды        Version: 3.0.0      State: Not installed                   
		        Install Method: Manual
	typology             Name: Типологий                      Version: 3.0.0      State: Enabled                          
		       Install Method: Marketplace
	order                Name: Управление заказами    Version: 2.7.6      State: Not installed                   
		        Install Method: Marketplace
	genericobject        Name: Управление объектами  Version: 2.11.1     State: Not installed                   
		        Install Method: Marketplace

</pre></details>

### Anything else?

On test site whis 10.0.2 that work. In Setup information 
GLPI_IDOR_EXPIRES: 7200
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: 
GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org
GLPI_INSTALL_MODE: TARBALL

**Comment by MisterSlam** [comment]
I`m add 2 directive, restart web server, and this is work.
P.S. directive "session.cookie_secure" and "session.cookie_httponly"

**Comment by cconard96** [comment]
Hello,

Can you test the fix in #12678?

**Comment by MisterSlam** [comment]
> Hello,
> 
> Can you test the fix in #12678?

Yes, that fix problem. Thank you.
